api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -15,11 +15,14 @@ import "envoy/config/trace/v3/http_tracer.proto";
 import "envoy/type/http/v3/path_transformation.proto";
 import "envoy/type/tracing/v3/custom_tag.proto";
 import "envoy/type/v3/percent.proto";
+import "envoy/extensions/transport_sockets/tls/v3/common.proto";
 
 import "google/protobuf/any.proto";
 import "google/protobuf/duration.proto";
 import "google/protobuf/wrappers.proto";
 
+import "xds/type/matcher/v3/matcher.proto";
+
```

Comment:
Please add `"@com_github_cncf_xds//xds/type/matcher/v3:pkg"` to the deps of this http_connection_manager.proto because you used matcher here.